packages/data/abstract/prophecies/lib/prophecies.dart

```diff
@@ -0,0 +1,4 @@
+library prophecies;
+
+export 'src/prophecies.dart';
+export 'src/json.dart';
```

Comment:
Dart natively supports json and this why I added json repository here.
I'm planning to rename abstract to native, to solve many confusions you had.
What about it?

Reply:
@bgoncharuck right. My point is that if Dart natively support json serialization, that does not mean that this is indeed an abstraction. For example, if Java have an implementation for File I/O, that does not mean, that file operations can be implemented in abstraction layer. Same with dart json. In this particular case, we have an abstraction layer, that just describes, how we'd like to see concrete implementations (like file storage or DB). DB does not require JSON serialization. Other types of storages might or might not require it as well. 
From the other hand, I can understand, that we need to use json serialization for debug purposes, I would rather use `toString()` method for pretty print.

I believe that still it's fine to keep json there, but with a thought that it is not because dart support it. 
I would vote for keeping this abstract/platform concepts with having clear division of abstraction that is visible to the outside and implementation that is usually hidden. We might not spend time on refactoring now, but lets keep it in mind.